app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  validates :name, presence: true
+  validates :name, length: { in: 3..15 }
+  validates :name, uniqueness: true
```

Comment:
You can write like this : 
`validates :name, presence: true, uniqueness: true, length: { in: 3..15 }`